implementations/rust/ockam/ockam/src/lib.rs

```diff
@@ -1,5 +1,7 @@
 //! Ockam is a library for building devices that communicate securely, privately
```

Comment:
We may want to update this line to be consistent with other taglines across Ockam's properties, e.g. "End-to-end encryption and mutual authentication for distributed applications." I think the tagline here is confusing because it speaks about "building devices".